.github/workflows/create-release-branch.yml

```diff
@@ -43,60 +42,50 @@ jobs:
         with:
           node-version: '14.x'
 
-      # Try get node_modules from cache
-      - name: Restore node_modules from cache
-        uses: actions/cache@v2
-        with:
-          path: common/temp/pnpm-store
-          key: ${{ runner.os }}-${{ hashFiles('common/config/rush/pnpm-lock.yaml') }}
-
-      # Install dependencies
-      - name: Install rush
-        run: npm install -g @microsoft/rush@5.47.0
-      - name: Install dependencies
-        run: rush install
-
-      # Bump package versions
-      - name: Bump package versions
-        run: node common/config/node_modules/beachball/bin/beachball bump
-
-      - name: Synchronize package version reported to telemetry
-        run: node common/scripts/sync-telemetry-package-version
-
-      # Important to check version consistency again after bumping versions.
-      - name: Ensure all package versions are consistent
-        run: rush ensure-consistent-versions
-
-      # Retrieve new version to create branch with
+      # get version number from communication react folder
       - name: Retrieve new version from package.json
         id: version
         run: |
           ver=$(jq -r .version packages/communication-react/package.json)
           echo version: $ver
           echo "::set-output name=version::$ver"
 
-      # Commit changes
+      # Create new release branch
       - name: Hop into new branch
         run: git checkout -b release/${{ steps.version.outputs.version }}
+
+      # Choose stable dependency for sdk
+      - name: Choose stable dependency for sdk if stable
+        if: ${{ startsWith(github.events.inputs.branch, 'prerelease-stable-minor/) || startsWith(github.events.inputs.branch, 'prerelease-stable-patch/)}}
+        run: node common/scripts/choose-stable-sdk-deps.js
+
+      - name: Choose beta dependency for sdk if beta
+        if: ${{ startsWith(github.events.inputs.branch, 'prerelease-beta/' )}}
+        run: node common/scripts/choose-beta-sdk-deps.js
+
+      # Synchronize the telemetry package versions.
+      - name: Synchronize package version reported to telemetry
+        run: node common/scripts/sync-telemetry-package-version
+
+      # we want to disable the beta CI here if we are doing a stable release.
+      # how should we do that?
+
+      # Commit new dependencies
       - name: Commit changes
         run: |
           git add .
-          git commit -m "Version bump"
+          git commit -m "Selected stable sdk's for release branch"
```

Comment:
Don't you have to push the release branch so that it shows up in GH?

Reply:
yes 😅